js/menu.js

```diff
@@ -0,0 +1,34 @@
+const headerDate = document.querySelector('.date');
+const list = document.getElementById('list-book');
+const contact = document.getElementById('contact');
+const addNew = document.getElementById('addNew');
+
+headerDate.innerHTML = Date();
+
+list.addEventListener('click', () => {
+  const books = JSON.parse(localStorage.getItem('books')) || [];
+  if (books.length === 0) {
+    document.querySelector('.list-book').innerHTML = '<p class="no-book"> No book add yet</p>';
+    document.querySelector('.all-awesome').style.display = 'none';
+    document.querySelector('.list-book').style.border = 'none';
+  } else {
+    document.querySelector('.all-awesome').style.display = 'block';
+    /* global display */
+    /* eslint no-undef: "error" */
```

Comment:
- [ ] please avoid turning off linters they are used to make your project work better and easier to read.